### PR TITLE
SSR-3415: Add HeaderVisible Property for Table

### PR DESCRIPTION
**Pull request by h000780**

### CODE REVIEW

File: src/table/test/headerVisible.test.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      expect(header.hasAttribute("hidden")).to.equal(false);
+    });
+    it("should be displayed none when assigned false in constructor", async () => {
+      const container = new Table({
+        columns: [...columns],
+        data: [...data],
+        headerVisible: false,
+      });
+      const el = await fixture(container);
+      const header = el.querySelector(
```
Comment (nmanhit) [inline src/table/test/headerVisible.test.ts:68]: Please change "header" variable to "tableHeader".
I get confused between HTML tag header "<header>"

File: docs/document/website/translated_docs/ja/version-1.12.0/components/desktop/table.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+#### Table
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| className | string | ""  | コンポーネントの class 名 | |
+| id | string | ""  | コンポーネントの id 名 | |
+| label | string | ""  | コンポーネントの説明ラベル | 未指定、あるいは空文字の場合、label は表示されない |
+| actionButton | boolean | true | 行追加/削除ボタンの表示/非表示設定 | |
+| headerVisible | boolean | true | テーブルヘッドの表示/非表示設定 | |
```
Comment (TomokoMiyake) [inline docs/document/website/translated_docs/ja/version-1.12.0/components/desktop/table.md:32]:
```suggestion
| headerVisible | boolean | true | ヘッダーの表示/非表示設定 | |
```

File: docs/document/docs/components/desktop/table.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ Here is a list of properties that can be used for modifying the component:
 | id | string | "" | Component id name | |
 | label | string | ""  | Label for the component | Label is not displayed if unspecified or empty |
 | actionButton | boolean | true | Show/Hide the add/remove row button | |
+| headerVisible | boolean | true | Show/Hide the table header | |
```
Comment (TomokoMiyake) [inline docs/document/docs/components/desktop/table.md:31]:
```suggestion
| headerVisible | boolean | true | Show/Hide the header | |
```

File: docs/document/website/versioned_docs/version-1.12.0/components/desktop/table.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+#### Table
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| className | string | ""  | Component class name | |
+| id | string | "" | Component id name | |
+| label | string | ""  | Label for the component | Label is not displayed if unspecified or empty |
+| actionButton | boolean | true | Show/Hide the add/remove row button | |
+| headerVisible | boolean | true | Show/Hide the table header | |
```
Comment (TomokoMiyake) [inline docs/document/website/versioned_docs/version-1.12.0/components/desktop/table.md:32]:
```suggestion
| headerVisible | boolean | true | Show/Hide the header | |
```

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
Please update document.

**Review by TomokoMiyake — APPROVED** [review]
LGTM!

**Pull request merged**